src/Quartz/Impl/StdSchedulerFactory.cs

```diff
@@ -508,11 +516,14 @@ protected virtual void ValidateConfiguration()
 
             Type tpType = loadHelper.LoadType(cfg.GetStringProperty(PropertyThreadPoolType)) ?? typeof(SimpleThreadPool);
 
+#if WINDOWS_THREADPOOL  // TODO (NetCore Port) - We can't use the ClrThreadPool on .NET Core because cross-plat threadpool implementations don't expose all the APIs we need
+                        // Use the SimpleThreadPool there for now and in the future create a new async-friendly threadpool that will work cross-plat.
             if (tpType == typeof(SimpleThreadPool))
```

Comment:
There is some discussion of the Min/MaxThreads issue here: https://github.com/dotnet/cli/issues/889#issuecomment-173331862.  

The simplest way to limit concurrency in .NET Core is probably to use SemaphoreSlim.  For example:

```csharp
    class ConcurrencyLimiter
    {
        private readonly SemaphoreSlim _sem;

        public ConcurrencyLimiter(int maxConcurrencyLevel)
        {
            _sem = new SemaphoreSlim(maxConcurrencyLevel);
        }

        public async Task Schedule(Func<Task> asyncOperation)
        {
            await _sem.WaitAsync();
            try
            {
                await asyncOperation();
            }
            finally
            {
                _sem.Release();
            }
        }
    }
```